src/workloads/transactions/LLTQueryLowLtc.yml

```diff
@@ -4,6 +4,14 @@ Description: |
   Workload to Benchmark the effect of LongLivedTransactions on a Query workload.
 
 GlobalDefaults:
+  # Start: The following block gets updated for the other configurations.
+  # In-memory: Database size works out about 12GB.
+  InitialDocumentCount: &InitialNumDocs 10000000
```

Comment:
Parameters were moved together to the top of the file. This is the source for all the other LLTQuery*.yml files.